SnakeGame/view.py

```diff
@@ -1,14 +1,49 @@
 import pygame
+from model import CellContent
+import enum
 
 class GameView(object):
     
-    DEFAULT_SIZE = width, length = 540, 540
     BG_COLOR = green = 0, 135, 0
+    SNAKE_COLOUR = 135, 0, 0
+    FOOD_COLOUR = 255, 51, 0
+    SNAKEHEAD_COLOR = 0, 0, 0 
 
-    def __init__(self):
-        self.size = self.DEFAULT_SIZE
+    def __init__(self, gridDimensions=(10, 10), gridSize=(540, 540)):
+        #Grid dimensions in number of cells
+        self.width = gridDimensions[0]
+        self.height = gridDimensions[1]
+        #grid dimensions in pixels
+        self.gridSize=gridSize
+        #grid cell dimensions in pixels
+        self.cellWidth = gridSize[0]/self.width
+        self.cellHeight = gridSize[1]/self.height
 
-    def init(self):
-        self.screen = pygame.display.set_mode(self.size)
+    def render(self, grid):
+        self.screen = pygame.display.set_mode(self.gridSize)
         self.screen.fill(self.BG_COLOR)
-        pygame.display.flip()
+        
+        for i in range(self.height):
+            for j in range(self.width):
+                if(grid[i][j] == CellContent.Empty):
+                    pygame.draw.rect(self.screen, self.BG_COLOR, (self.getleftCoord(j,0), self.getTopCoord(i,0), self.cellWidth, self.cellHeight))
+                elif (grid[i][j] == CellContent.SnakeBody):
+                    pygame.draw.rect(self.screen, self.SNAKE_COLOUR, (self.getleftCoord(j, 0.05), self.getTopCoord(i, 0.05), self.cellWidth*0.9, self.cellHeight*0.9))
+                elif (grid[i][j] == CellContent.SnakeHead):
+                     pygame.draw.rect(self.screen, self.SNAKEHEAD_COLOR, (self.getleftCoord(j, 0.1), self.getTopCoord(i, 0.1), self.cellWidth*0.8, self.cellHeight*0.8))
+                elif(grid[i][j] == CellContent.Food):
+                    pygame.draw.ellipse(self.screen, self.FOOD_COLOUR, (self.getleftCoord(j, 0.25), self.getTopCoord(i, 0.25), (self.cellWidth)/2.0, (self.cellHeight)/2.0))
```

Comment:
These work great! I would, however, define constants at the top for these values so that we can tweak them in one place to see how the game looks with different values (it would also be less error-prone).